compiler/rustc_smir/src/stable_mir/ty.rs

```diff
@@ -17,6 +17,7 @@ type Span = Opaque;
 #[derive(Clone, Debug)]
 pub enum TyKind {
     RigidTy(RigidTy),
+    Alias(AliasDef, GenericArgs, AliasKind),
```

Comment:
I'd make this match more closely what we have in `rustc_middle`. So `Alias(AliasKind, AliasTy)` and `AliasTy` has the `AliasDef` and `GenericArgs`. No big reasons for this, it's just to follow the pattern we have for other variants.